File: src/main/java/top/zbeboy/isy/web/vo/data/nation/NationVo.java
package top.zbeboy.isy.web.vo.data.nation;

import lombok.Data;

import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by lenovo on 2016-11-02.
 */
@Data
public class NationVo {
    private Integer nationId;
    @NotNull
    @Size(max = 30)
    private String  nationName;
}

File: src/main/java/top/zbeboy/isy/web/vo/data/nation/NationVo.kt
package top.zbeboy.isy.web.vo.data.nation

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2017-12-12 .
 **/
open class NationVo {
    var nationId: Int? = null
    @NotNull
    @Size(max = 30)
    var nationName: String? = null
}